src/routes/authRoutes.js

```diff
@@ -5,6 +5,6 @@ const { register, login, logout } = require('../controllers/authController');
 
 router.post('/register', register);
 router.post('/login', login);
-router.post('/logout', logout);
+router.get('/logout', logout);
```

Comment:
I think this got lost from a previous PR; do you mind switching this back to a `post` ?

Reply:
my mistake, switching this back to a post.